content/blog/2020/09/2020-09-09-jenkins-continuous-evolution-cdcon.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Learn more about Jenkins' continuous evolution at CDCon"
+tags:
+- cdcon
+- talks
+- jcasc
+- cicd
+- tekton
+author: cdfoundation
```

Comment:
The author entry needs a biography.  I pushed one as a prototype so that you can use it as a basis for change or correction.,
```suggestion
author: cdfoundation
```

Reply:
If you decide to accept that author bio that I added, then this is how the page will look:

![screencapture-localhost-4242-blog-2020-09-09-jenkins-continuous-evolution-cdcon-2020-09-09-16_13_16](https://user-images.githubusercontent.com/156685/92660326-712e0600-f2b7-11ea-87e9-4c3eab8c667f.png)

If you'd like to provide an image for the CDF biography, that can be added as part of the bio.  Refer to other biographical entries for the format.